src/RoslynAnalyzers/Microsoft.CodeAnalysis.Analyzers/Microsoft.CodeAnalysis.Analyzers.md

```diff
@@ -205,7 +205,7 @@ A CodeFixProvider should provide FixAll support to enable users to fix multiple
 |Category|Correctness|
 |Enabled|True|
 |Severity|Warning|
-|CodeFix|True|
+|CodeFix|False|
```

Comment:
very strange.  i don't know how this doc generator works, or why it now thinks there is no codefix for some of these analyzers.

Reply:
i wonder if it is this awful code:

![image](https://github.com/user-attachments/assets/cf739402-1933-4c2b-a166-8db2f933af1e)

Reply:
Spoke offline and I suggested it might be the transitive pinning in conjunction with the source-build specific PackageVersion removal in the GenerateDocumentationAndConfigFiles tool project. This led us to add a Directory.Packages.props to disable pinning. It apparently isn't the entire fix, but did resolve one of the reported errors.